pgbouncer/tests/test_pgbouncer_unit.py

```diff
@@ -8,15 +8,6 @@
 from datadog_checks.pgbouncer import PgBouncer
 
 
-@pytest.mark.unit
```

Comment:
I think this is more of an integration test rather than a unit test, since it's running through the entire check and expecting a failing service check. Thoughts?